portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,90 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>Utibeno Ekpoudom</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
+    <link
+      href="https://fonts.googleapis.com/css?family=Merienda|Neucha&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <section class="hero">
+      <img src="images/portfolio_pic.jpg" alt="Uty" />
```

Comment:
I would expect to see this image being added to your repo in this commit. Without that, it won't be possible to fully publish your site.

Reply:
just added the images folder